lib/matplotlib/backends/backend_qt5.py

```diff
@@ -1019,6 +1031,80 @@ def trigger(self, *args, **kwargs):
 backend_tools.ToolCopyToClipboard = ToolCopyToClipboardQT
 
 
+@cbook.deprecated("3.0")
+def error_msg_qt(msg, parent=None):
+    if not isinstance(msg, str):
+        msg = ','.join(map(str, msg))
+
+    QtWidgets.QMessageBox.warning(None, "Matplotlib",
+                                  msg, QtGui.QMessageBox.Ok)
+
+
+@cbook.deprecated("3.0")
+def exception_handler(type, value, tb):
```

Comment:
Remove.

Reply:
I think this function is being accidentally resurrected?